trello_items.py

```diff
@@ -0,0 +1,127 @@
+import requests
+import os
+from datetime import datetime
+import logging as log
+
+BASE_URL = 'https://api.trello.com/1'
+
+# the trello board ID to use for all the lists and cards
+BOARD_ID = os.environ.get('TRELLO_BOARD_ID')
```

Comment:
Shouldn't you be reusing the config class here? (& for the 4 other examples below)